### PR TITLE
home page: switch sponsors-list from foundation grid to block-grid

### PR DESCRIPTION
**Pull request by MaxWofford**

### CODE REVIEW

**Comment by zachlatta** [comment]
Can you squash the two commits into one? See
https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request#squash-your-changes-optional.

--
Zach Latta
Executive Director of hackEDU[https://hackedu.us/]
http://zachlatta.com
https://github.com/zachlatta
https://twitter.com/zachlatta


On Thu, Apr 2, 2015, at 08:11 PM, Max Wofford wrote:
> 
> You can view, comment on, or merge this pull request online at:


> https://github.com/hackedu/hackedu/pull/73


> Commit Summary


> * home page: switch sponsors-list from foundation grid to block-grid
> File Changes


> * *M* app/assets/stylesheets/pages.css.scss[https://github.com/hackedu/hackedu/pull/73/files#diff-0]
    (9)
> * *M* app/views/pages/home.html.haml[https://github.com/hackedu/hackedu/pull/73/files#diff-1]
    (82)

**Comment by zachlatta** [comment]
Can you merge master into this so there's no merge conflict?

**Comment by MaxWofford** [comment]
@zachlatta

**Comment by zachlatta** [comment]
> and then squash your changes after merging master in

File: app/views/pages/home.html.haml [hunk 1]
```diff
@@ -67,6 +67,36 @@
...
+        %img{alt: 'nitrous.io', src: '/assets/nitrousio.svg'}
+    %li
+      = link_to "http://www.rackspace.com/" do
+        %img{alt: 'Rackspace', src: '/assets/rackspace.svg'}
+    %li
+      = link_to "https://codehs.com/" do
+        %img{alt: 'CodeHS', src: '/assets/codehs.png'}
+    %li.last
+      = link_to "http://firstcodeacademy.com/" do
+        %img{alt: 'First Code Academy', src: '/assets/first_code_academy.png'}
```
Comment (zachlatta) [inline app/views/pages/home.html.haml]: Will using the raw `img` tag work with the asset pipeline in production? We can use the `image_tag` (http://api.rubyonrails.org/classes/ActionView/Helpers/AssetTagHelper.html#method-i-image_tag) which will link to the correct, renamed asset in production.

**Comment by MaxWofford** [comment]
YES! IT WORKED! @zachlatta _now_ it is ready for merge.

**Pull request merged**